designs/sandy/src/index.mjs

```diff
@@ -10,4 +11,4 @@ const Sandy = new Design({
 })
 
 // Named exports
-export { skirt, waistband, Sandy }
+export { skirt, waistband, draftRingSector, Sandy }
```

Comment:
// I don't know the answers either, so below just guesses based on my experience as a software dev; will defer to anyone with a stronger opinion.

1. It preferably should be documented, but I have no idea where. The function itself is relatively well commented, maybe that is enough?
2. Probably not, the license lets us do much more with the code if we want to and making one function exported is absolutely tiny. As for politeness, if they are still around they'll see it anyway, and if they aren't they are extremely unlikely to care.
3. No, having it in a separate "library" (or whatever the equivalent here is) would be best, but I don't have enough understanding of neither javascript nor this codebase to be able to do that properly. ^^"
4. Copying code sounds much worse than importing it, especially since this is very generic code and it does _exactly_ the desired thing.
5. I think I tried this form of import and it didn't work (somethingsomethingoutofmodule?), but see my comment above about my lack of JS knowledge. :P